### PR TITLE
Feature/df 479 responsible rav

### PR DESCRIPTION
**Pull request by 99999989**

### CODE REVIEW

File: alv-portal-ui/src/app/job-publication/job-publication.component.html [hunk 1]
```diff
@@ -40,15 +40,21 @@ <h2 cl>
...
+      <li class="mb-1" *ngIf="responsibleJobCenter$ | async as jobCenter">
         {{'portal.job-publication.submit.info.item3' | translate}}
         <p class="mt-3">
-          RAV Aussendienst<br>
-          Hochstrasse 37<br>
-          4002 Basel<br>
-          Tel.: +41612675033<br>
-          Fax: +41612675052<br>
-          Email: jobroom.rav@bs.ch
+          {{jobCenter.address?.name}}<br>
```
Comment (pavax) [inline alv-portal-ui/src/app/job-publication/job-publication.component.html]: isn't there a address component that we could re-use?

Comment (pavax) [reply alv-portal-ui/src/app/job-publication/job-publication.component.html]: copied from job-center-context component:

```html
  <alv-post-address [address]="{
                        name: address.name,
                        street: address.street,
                        houseNumber: address.houseNumber,
                        postalCode: address.zipCode,
                        city: address.city}">
  </alv-post-address>
```

File: alv-portal-ui/src/app/job-publication/job-publication-form/job-publication-form.component.ts [hunk 2]
```diff
@@ -114,7 +115,7 @@ export class JobPublicationFormComponent extends AbstractSubscriber implements O
     const createJobAdvertisement = jobPublicationFormMapper.mapToCreateJobAdvertisement(this.jobPublicationForm.value, 'de');
     this.jobAdvertisementRepository.save(createJobAdvertisement)
       .subscribe((resp) => {
-        this.jobPublicationCreated.emit();
+        this.jobPublicationCreated.emit(resp.json);
```
Comment (pavax) [inline alv-portal-ui/src/app/job-publication/job-publication-form/job-publication-form.component.ts]: why do we need to specify resp.json? should the method save just return us a JobAdvertisement

File: alv-portal-ui/src/app/job-publication/job-publication.component.html [hunk 3]
```diff
@@ -40,15 +40,25 @@ <h2 cl>
...
-          Email: jobroom.rav@bs.ch
+          <alv-post-address [address]="{
+                        name: jobCenter.address?.name,
+                        street: jobCenter.address?.street,
+                        houseNumber: jobCenter.address?.houseNumber,
+                        postalCode: jobCenter.address?.zipCode,
+                        city: jobCenter.address?.city}">
+          </alv-post-address>
+          <ng-container *ngIf="jobCenter.phone">
+            {{'portal.job-publication.submit.info.phone' | translate}}: {{jobCenter.phone}}<br>
```
Comment (pavax) [inline alv-portal-ui/src/app/job-publication/job-publication.component.html]: shouldn't we use: 
```html
<a class="link" href="tel:{{jobCenter.phone}}">{{jobCenter.phone | phoneNumber}}</a>
```

File: alv-portal-ui/src/app/job-publication/job-publication.component.html [hunk 4]
```diff
@@ -40,15 +40,25 @@ <h2 cl>
...
+          </ng-container>
+          <ng-container *ngIf="jobCenter.fax">
+            {{'portal.job-publication.submit.info.fax' | translate}}: {{jobCenter.fax}}<br>
+          </ng-container>
+          <ng-container *ngIf="jobCenter.email">
+            {{'portal.job-publication.submit.info.email' | translate}}: {{jobCenter.email}}
```
Comment (pavax) [inline alv-portal-ui/src/app/job-publication/job-publication.component.html]: shoudln't we use:
```html
<a class="link" href="mailto:{{jobCenter.email}}">{{jobCenter.email}}</a>
````

File: alv-portal-ui/src/app/job-publication/job-publication-form/job-publication-form.component.ts (hunk 2 above)
Comment (99999989) [reply alv-portal-ui/src/app/job-publication/job-publication-form/job-publication-form.component.ts]: fixed

File: alv-portal-ui/src/app/job-publication/job-publication.component.html (hunk 3 above)
Comment (99999989) [reply alv-portal-ui/src/app/job-publication/job-publication.component.html]: Fixed

File: alv-portal-ui/src/app/job-publication/job-publication.component.html (hunk 4 above)
Comment (99999989) [reply alv-portal-ui/src/app/job-publication/job-publication.component.html]: fixed

File: alv-portal-ui/src/app/job-publication/job-publication.component.html (hunk 1 above)
Comment (99999989) [reply alv-portal-ui/src/app/job-publication/job-publication.component.html]: Fixed

**Pull request merged**